lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -136,6 +132,63 @@ import qualified Data.ByteString as BS
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as T
 
+-- @AddressScheme@ correlates @key@s with @Addresses@.
+--
+-- In a @SeqState@ we need tell if an @Address@ was created using a given @key@.
+--
+-- A @key@ may corrspond to multiple @Address@es. We cannot* convert
+-- an address into a public key.
```

Comment:
These two statements are false. Jörmungandr does embed the full public key, so it's possible to go back and forth. And, a key itself correspond to only a single address; unless you meant that a key can be part of many addresses when grouped with some other key or script hash.

Reply:
> Jörmungandr does embed the full public key

Hence the `*` asterisk :)